carabiner-to-py/CarabinerToPy.py

```diff
@@ -0,0 +1,80 @@
+import socket
+import edn_format
+import time
+
+class LinkListener():
+    """A simple python client to communicate with carabiner (a Abelton Link connector)
+    Carabiner server must be running to use. Requires edn_format [$pip install edn_format]"""
+    def __init__(self, tcp_ip = '127.0.0.1', tcp_port = 17000, buffer_size = 1024):
+        self.tcp_ip = tcp_ip
+        self.tcp_port = tcp_port
+        self.buffer_size = buffer_size
+        self.s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.s.connect((tcp_ip, tcp_port))
+
+    def decode_edn_msg(self, msg):
+        """Decodes a TCP message from Carabiner to python dictionary"""
+        msg = msg.decode()
+        striped_msg = msg[msg.index('{'):]
+        decoded_msg = edn_format.loads(striped_msg)
+
+        # Because the edf_format package does not return normal dam dicts (or string keywords). What dicts.
+        if type(decoded_msg) is edn_format.immutable_dict.ImmutableDict:
+            decoded_dict = {}
+            for key, value in decoded_msg.dict.items():
+                print()
+                decoded_dict[str(key).strip(':')] = value
+            decoded_msg = decoded_dict
+
+        return decoded_msg
+
+    def status(self):
+        """Wrapper for Status"""
+        self.s.send(b'status')
+        return_msg = self.s.recv(self.buffer_size)
+        return self.decode_edn_msg(return_msg)

```

Comment:
https://github.com/bdyetton/link-to-py/tree/master
I decided this might be better as its own repo, can you mention this in your README please.

Thanks!